src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    display: block;
+    background-size: cover;
+    background-position: center;
+
+    margin: 0 auto 40px;
+  }
+
+  &__tittle {
+    display: flex;
+    align-items: center;
+    color: #060B35;
+    font-size: 12px;
+    line-height: 18px;
+
+    margin-bottom: 4px;
+  }
+
+  &__product_code {
+    color: #616070;
+    font-size: 10px;
+    font-style: normal;
+
+    font-weight: 400;
+    line-height: 14px;
+
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    color: #060B35;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-left: 8px;
+  }
+
+  &__prise {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+
+     &_text {
+      color: #616070;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 18px;
+     }
+
+     &_number {
+      font-size: 16px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__buy {
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead

Reply:
@Viktor-Kost